guides/keras_cv/object_detection_keras_cv.py

```diff
@@ -17,15 +17,27 @@
 models!
 
 Let's give KerasCV's object detection API a spin.
+
+With Keras Core, we can use TensorFlow, JAX, or PyTorch as our backend for
+KerasCV! Try using "jax" or "torch" as the KERAS_BACKEND and re-running this
+guide to experiment with different backends.
 """
 
 """shell
-!pip install --upgrade git+https://github.com/keras-team/keras-cv
+!pip install -U keras-core git+https://github.com/keras-team/keras-cv.git
 """
-import tensorflow as tf
+import os
+
+os.environ["KERAS_BACKEND"] = "tensorflow"
+
+from tensorflow import data as tf_data
```

Comment:
Why not `tf.data`?

Reply:
The idea was to demonstrate that TF isn't used except for tf data, but I switched it back because this is a departure from normal convention